js/components/file-upload.tsx

```diff
@@ -1,37 +1,17 @@
 ﻿import * as React from 'react';
-import { findDOMNode } from 'react-dom';
-// tslint:disable-next-line
-import Dropzone from 'react-dropzone';
 import registeredComponents from '../constants/registeredComponents';
 import IFileUploadProps from '../interfaces/IFileUploadProps';
 import outcome from './outcome';
 import '../../css/files.less';
+import FileUpload from '@boomi/react-file-upload';
 
-interface IFileStatus {
-    fileName: string;
-    uploadSuccessful: boolean;
-}
-
-interface IFileUploadState {
-    fileStatusList?: IFileStatus[];
-    isUploadDisabled?: boolean;
-    isFileSelected?: boolean;
-    isProgressVisible?: boolean;
-    isUploadComplete?: boolean;
-    fileNames?: string[];
-    progress?: number;
-    error?: any;
-    files?: any;
-}
-
-class FileUpload extends React.Component<IFileUploadProps, IFileUploadState> {
+class FileUploadWrapper extends React.Component<IFileUploadProps> {
```

Comment:
This component no longer has state and no lifecycle methods so could be converted to a function component.

Reply:
Ah yes, good idea, done it now